catalyst/dl/callbacks/mixup.py

```diff
@@ -1,50 +1,56 @@
 from typing import List
-from catalyst.dl.core.state import RunnerState
-from catalyst.dl.callbacks import CriterionCallback
+
 import numpy as np
+
 import torch
 
+from catalyst.dl.core.state import RunnerState
+from catalyst.dl.callbacks import CriterionCallback
+
 
 class MixupCallback(CriterionCallback):
-    def __init__(self, fields: List[str] = ('features',), alpha=1.0,
+    def __init__(self, fields: List[str] = ("features",), alpha=1.0,
```

Comment:
add docs in [Google format](https://github.com/catalyst-team/catalyst/blob/master/CONTRIBUTING.md#documentation) please

Reply:
Thanks, done.